src/components/my-career/data/hooks.jsx

```diff
@@ -3,29 +3,28 @@ import { logError } from '@edx/frontend-platform/logging';
 import { camelCaseObject } from '@edx/frontend-platform/utils';
 import Plotly from 'plotly.js-dist';
 
-import { getLearnerSkillLevels, getLearnerSkillQuiz } from './service';
+import { getLearnerSkillLevels, getLearnerProfileInfo } from './service';
 import { getSpiderChartData, prepareSpiderChartData } from './utils';
 
-export function useLearnerSkillQuiz(username) {
-  const [learnerSkillQuiz, setLearnerSkillQuiz] = useState();
+export function useLearnerProfileData(username) {
+  const [profileData, setLearnerProfileData] = useState();
```

Comment:
nit: can you rename `profileData` to `learnerProfileData`.

Reply:
Done